pyvista/plotting/plotting.py

```diff
@@ -1508,6 +1524,11 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
 
         rgb = kwargs.pop('rgba', rgb)
 
+        # account for legacy behavior
+        if 'stitle' in kwargs:  # pragma: no cover
+            warnings.warn(USE_SCALAR_BAR_ARGS)
```

Comment:
Do we want to omit an explicit `DeprecationWarning` instead?

Reply:
Fixed in d0846067a565bc6eed0cd4fc5517ef2629c559d9